Auth0/CredentialsManager.swift

```diff
@@ -40,14 +40,25 @@ public struct CredentialsManager {
         self.authentication = authentication
     }
 
-    /// Enable TouchID Authentication for additional securtity during credentials retrieval.
+    /// Enable Face ID / Touch ID Authentication for additional securtity during credentials retrieval.
     ///
     /// - Parameters:
     ///   - title: main message to display in TouchID prompt
     ///   - cancelTitle: cancel message to display in TouchID prompt (iOS 10+)
     ///   - fallbackTitle: fallback message to display in TouchID prompt after a failed match
+    @available(*, deprecated, message: "see enableBioAuth(withTitle title:, cancelTitle:, fallbackTitle:)")
     public mutating func enableTouchAuth(withTitle title: String, cancelTitle: String? = nil, fallbackTitle: String? = nil) {
-        self.touchAuth = TouchAuthentication(authContext: LAContext(), title: title, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle)
+        self.bioAuth = BioAuthentication(authContext: LAContext(), title: title, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle)
+    }
+
+    /// Enable Device Biometric Authentication for additional securtity during credentials retrieval.
+    ///
+    /// - Parameters:
+    ///   - title: main message to display in TouchID prompt
```

Comment:
parameters talk about "touch" id. Should be changed to "bio" something

Reply:
Well it is only used during a touch prompt, it's not valid for face id. could be more verbose perhaps 'display when using touch id'

Reply:
I don't know how this works as haven't tested myself. So when a face is expected no message is displayed, nothing that can be customized?

Reply:
No, it just comes up with this standard image / text.  https://support.apple.com/library/content/dam/edam/applecare/images/en_US/ios11-iphone-x-wallet-apple-pay-animation.gif